### DISCUSSION TITLE
Inability to select text vertically

### DISCUSSION THREAD

**Discussion by a787800255** [dbeaver/dbeaver#19438] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I often find it difficult to select and edit specific columns of text in the SQL Editor, especially when dealing with large and complex SQL queries. The inability to select text vertically (i.e., column-wise) can be quite frustrating when trying to make precise modifications to the code.

**Describe the solution you'd like**
I would like to request a new feature that allows users to select text vertically (column-wise) in the SQL Editor by holding down the `Alt` key and using the left mouse button to click and drag. This would enable users to easily select and edit columns of text without affecting the surrounding code.

**Describe alternatives you've considered**
One alternative I've considered is using an external text editor with column-wise selection capabilities, such as Notepad++ or Visual Studio Code, and then copying the modified code back into the SQL Editor. However, this workaround is cumbersome and inefficient, as it requires switching between different applications and increases the risk of errors.

**Additional context**
The ability to select text vertically is a common feature in many code and text editors, and it would greatly improve the user experience in the SQL Editor by allowing for more precise and efficient editing. 

The screenshoot of SQL Editor:
![image](https://user-images.githubusercontent.com/32060702/226237218-708372ee-77f3-4526-a52c-5c7594ab8809.png)

The screenshoot of Notepad++:
![image](https://user-images.githubusercontent.com/32060702/226237280-e096cca4-4e09-46d3-98d8-febc7d8cbb59.png)

**Comment by ShadelessFox** [comment, marked as answer]
Hello @a787800255,

Have you tried using block selection? It can be toggled in the <kbd>Edit</kbd> &rArr; <kbd>Toggle Block Selection</kbd>, or by using shortcut <kbd>Alt+Shift+A</kbd>.

Please let me know if it solves your issue.

**Comment by a787800255** [comment]
thanks soooo much!!

**Comment by a787800255** [comment]
@ShadelessFox Really appreciate，the problem has been solved.
